### DISCUSSION TITLE
Dashboard: Automatically trigger refresh when exiting panel edit mode after having made variable changes

### DISCUSSION THREAD

**Discussion by bluefish6** [grafana/grafana#38176] [Dashboards, Panels & Visualizations]
**What happened**: After editing the value of some dashboard variable (time interval in my case) while being inside single-graph view, the dashboard graphs are not updated until the refresh is triggered manually or via "update every".

![ezgif-6-223b627227ef](https://user-images.githubusercontent.com/3324881/98660982-36435f80-2346-11eb-85bb-2719b164fa15.gif)

**What you expected to happen**: After exiting the single-graph view with updated value, the whole dashboard is instantly refreshed to reflect the new variable selection, without needing to click "refresh" or to wait until the auto-update is triggered.

**How to reproduce it (as minimally and precisely as possible)**:

- Go to a dashboard
- Set dashboard-wide variable like "time interval" to "1 day"
- Enter edit mode of single graph
- Change the variable to for example "1 minute"
- Exit edit mode of single graph (go back to whole dashboard view)
- Observe not updated graphs apart from the one which you edited just now

**Anything else we need to know?**:
If I remember correctly, this used to work as expected in grafana 5.x

**Environment**:
- Grafana version: Grafana v7.2.0 (efe4941ee3)
- Data source type & version: influxdb, version unknown (but doesn't seem to matter)
- OS Grafana is installed on: debian
- User OS & Browser: Ubuntu 20.04, Chromium  86.0.4240.111 (Official Build) snap (64-bit)
- Grafana plugins: unknown
- Others: ---

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by hugohaggmark** [comment]
This feature was implemented in https://github.com/grafana/grafana/pull/34097 and released in Grafana 8.0.0-beta2.
